### PR TITLE
Share content_ids for similar HTML Pubs

### PR DESCRIPTION
**Pull request by boffbowsh**
HTML Publications should have the same content_id across parent editions

This copies the content_id between attachments and adds a data
migration to back-populate the content_id. This is keyed on document_id
and attachment slug.

### CODE REVIEW

File: db/data_migration/20160527170207_share_content_ids_for_similar_html_attachments.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+HtmlAttachment.connection.execute(<<-SQL)
```
Comment (tvararu) [inline db/data_migration/20160527170207_share_content_ids_for_similar_html_attachments.rb]: I've seen this in `html_publication_sync_check.rb` as well, so I assume it's valid, but can you really close the argument paranthesis and then follow on the next line with the actual SQL?

**Comment by steventux** [comment]
:+1: I assume the migration SQL works :vhs:

**Comment by tvararu** [comment]
👍  From me as well.

File: db/data_migration/20160527170207_share_content_ids_for_similar_html_attachments.rb (hunk 1 above)
Comment (boffbowsh) [inline db/data_migration/20160527170207_share_content_ids_for_similar_html_attachments.rb]: Yep :)

**Pull request merged**